doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -35,6 +35,46 @@ Ubuntu 16.04.4 LTS was used throughout this document, other versions such as
      sudo service ssh status
      sudo service ssh start
 
+Install build tools and dependencies
+************************************
+
+Install development tools for ARCN development:
+
+* On a Ubuntu development system:
+
+  .. code-block:: none
+
+     sudo apt install gcc \
+     git \
+     make \
+     gnu-efi \
+     libssl-dev \
+     libpciaccess-dev \
+     uuid-dev \
+     libsystemd-dev \
+     libevent-dev \
+     libxml2-dev \
+     libusb-1.0-0-dev \
+     python3 \
+     python3-pip \
+     libblkid-dev \
+     e2fslibs-dev \
```

Comment:
delete the trailing continuation``\`` on this line (the next line is a new command)